In [6]:
#入力受け取り
print("input N >> ",end = "")
N = int(input())
print(N)

#ファイルの読み込み
f = open("popular-names.txt","r")

#一行ずつ読み込んで表示する
#改行処理が重複するのでtxtファイルから読み込んだ分は削除しておく
for i in range(N):
    readed_line = f.readline().strip("\n")
    print(readed_line)

f.close()

input N >> 5
Mary	F	7065	1880
Anna	F	2604	1880
Emma	F	2003	1880
Elizabeth	F	1939	1880
Minnie	F	1746	1880


--UNIXコマンドによる確認画面--
・-n(op) 表示する行数を指定

shota@LAPTOP-7H1S837B:~/py_venvs/nlp100/100knock2024/shota/chapter02$ head -n 5 popular-names.txt
Mary    F       7065    1880
Anna    F       2604    1880
Emma    F       2003    1880
Elizabeth       F       1939    1880
Minnie  F       1746    1880